boards/arm/nrf9131ek_nrf9131/board.cmake

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_BOARD_NRF9131EK_NRF9131_NS)
+  set(TFM_PUBLIC_KEY_FORMAT "full")
+endif()
+
+if(CONFIG_TFM_FLASH_MERGED_BINARY)
+  set_property(TARGET runners_yaml_props_target PROPERTY hex_file tfm_merged.hex)
+endif()
+
+# TODO: change to nRF9131_xxAA when such device is available in JLink
+board_runner_args(jlink "--device=nRF9160_xxAA" "--speed=4000")
```

Comment:
It looks like support for nRF9131 and nRF9161 was added in [v7.92m](https://www.segger.com/downloads/jlink/ReleaseNotes_JLink.html).